src/client/app/components/admin/PreferencesComponent.tsx

```diff
@@ -213,105 +224,99 @@ export default function PreferencesComponent() {
+									checked={localAdminPref.defaultChartToRender === chartType}
+								/>
+								{translate(chartType)}
+							</label>
+						</div>
+					))
+				}
 			</div>
-			<div style={bottomPaddingStyle}>
-				<p style={titleStyle}>
-					{`${translate('default.meter.maximum.errors')}:`}
-				</p>
-				<Input
-					type='number'
-					value={localAdminPref.defaultMeterMaximumErrors}
-					onChange={e => makeLocalChanges('defaultMeterMaximumErrors', e.target.value)}
-					maxLength={50}
-				/>
+			<p className='mt-2' style={labelStyle}>
+				<FormattedMessage id='default.graph.settings' />:
+			</p>
+			<div className='checkbox'>
+				<label>
+					<input
+						type='checkbox'
+						style={{ marginRight: '10px' }}
+						onChange={e => makeLocalChanges('defaultBarStacking', e.target.checked)}
+						checked={localAdminPref.defaultBarStacking}
+					/>
+					{translate('default.bar.stacking')}
+				</label>
 			</div>
-			<div style={bottomPaddingStyle}>
-				<p style={titleStyle}>
-					{`${translate('default.meter.disable.checks')}:`}
-				</p>
-				<Input
-					type='select'
-					value={localAdminPref.defaultMeterDisableChecks?.toString()}
-					onChange={e => makeLocalChanges('defaultMeterDisableChecks', e.target.value)}>
-					{Object.keys(TrueFalseType).map(key => {
-						return (<option value={key} key={key}>{translate(`TrueFalseType.${key}`)}</option>);
-					})}
-				</Input>
+			<div className='checkbox'>
+				<label>
+					<input
+						type='checkbox'
+						style={{ marginRight: '10px' }}
+						onChange={e => makeLocalChanges('defaultAreaNormalization', e.target.checked)}
+						checked={localAdminPref.defaultAreaNormalization}
+					/>
+					{translate('default.area.normalize')}
+
+				</label>
 			</div>
-			<div style={bottomPaddingStyle}>
-				<p style={titleStyle}>
-					<FormattedMessage id='default.help.url' />:
+			<div>
+				<p className='mt-2' style={labelStyle}>
+					{translate('default.area.unit')}
+
 				</p>
-				<Input
-					type='text'
-					value={localAdminPref.defaultHelpUrl}
-					onChange={e => makeLocalChanges('defaultHelpUrl', e.target.value)}
-				/>
+				<div className='radio'>
+					<label>
+						<input
+							type='radio'
+							name='areaUnitType'
+							style={{ marginRight: '10px' }}
+							value={AreaUnitType.feet}
+							onChange={e => makeLocalChanges('defaultAreaUnit', e.target.value)}
+							checked={localAdminPref.defaultAreaUnit === AreaUnitType.feet}
+						/>
+						{translate('AreaUnitType.feet')}
+					</label>
+				</div>
+				<div className='radio'>
+					<label>
+						<input
+							type='radio'
+							name='areaUnitType'
+							style={{ marginRight: '10px' }}
+							value={AreaUnitType.meters}
+							onChange={e => makeLocalChanges('defaultAreaUnit', e.target.value)}
+							checked={localAdminPref.defaultAreaUnit === AreaUnitType.meters}
+						/>
+						{translate('AreaUnitType.meters')}
+					</label>
+				</div>
 			</div>
+
+
```

Comment:
Sorry to just notice this now but on all the modal pages the submit and other buttons are to the right. I think one should be too for consitency. To work well, I think the entire page width should be limited since it can be made very wide and that only puts the drop down items way to the right. This would also put the buttons in a reasonable place on the right. You can either do this or add to the issue you opened on the discard button since related.

Reply:
Thanks for the comments. I have addressed these changes in commit [93a3575](https://github.com/OpenEnergyDashboard/OED/pull/1324/commits/93a3575b4e943ccf0d30376e562c26f8b588dca3). Please let me know what you think, thanks!